contracts/mock/MockStrategy2.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.6.8;
+
+import "@openzeppelinV3/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelinV3/contracts/math/SafeMath.sol";
+import "@openzeppelinV3/contracts/math/SafeMath.sol";
+import {BaseStrategy} from "@yearnvaults/contracts/BaseStrategy.sol";
+
+/*
+ * MockStrategy
+ */
+
+contract MockStrategy2 is BaseStrategy {
```

Comment:
I couldn't find a better name for a mock strategy which can set an acceptable call cost :)